### PR TITLE
Update managing-users.md for subaccount users

### PR DESCRIPTION
**Pull request by haleysolomon**
Added info about subaccount reporting users.

### CODE REVIEW

File: support/user-guide/managing-users.md [hunk 1]
```diff
@@ -30,3 +30,12 @@ Admin users can add new users to their account by from the [Manage Users](https:
 5. The new user will receive an email to join your account similar to the one below.
 
 	![](media/user-management/invitation-email.png)
+	
+### Subaccount Users
+*Limited access. Reach out to our support team or your Technical Account Manager to have this option enabled for your account.
+
+Reporting Users can optionally be assigned to a specific subaccount. Check the "assign to subaccount" box when creating the user and then select the subaccount. Learn more about subaccounts and subaccount users [here](https://www.sparkpost.com/docs/user-guide/subaccounts/).
+
+Only Reporting Users support subaccount designation at this time.
```
Comment (jgzamora) [inline support/user-guide/managing-users.md]: designation -> assignment?

File: support/user-guide/managing-users.md [hunk 2]
```diff
@@ -30,3 +30,12 @@ Admin users can add new users to their account by from the [Manage Users](https:
...
+
+Subaccount UI access is available to use for free internally to your company and can be made available to your customers via a reseller agreement.
```
Comment (jgzamora) [inline support/user-guide/managing-users.md]: "to your company" -> "within your company"

File: support/user-guide/managing-users.md (hunk 1 above)
Comment (haleysolomon) [reply support/user-guide/managing-users.md]: updated

File: support/user-guide/managing-users.md (hunk 2 above)
Comment (haleysolomon) [reply support/user-guide/managing-users.md]: updated

**Pull request merged**